shared/common-adapters/clickable-box.js.flow

```diff
@@ -17,6 +17,8 @@ export type Props = {
   activeOpacity?: number,
   // desktop only
   onMouseOver?: ?(event: SyntheticEvent<>) => void,
+  // You must specify both onMouseEnter and onMouseLeave if you
+  // specify one.
```

Comment:
why is this true

Reply:
Otherwise, one of `_onMouseEnter` or `_onMouseLeave` won't fire, so the `mouseIn` state will get stuck in one state or the other. That's why I asked if there was a better way above.

Reply:
this doesn't make sense to me, if you assign those values then this state never gets set ever now

Reply:
Yeah, I think my changes were incorrect. Removed this comment.